### PR TITLE
add transifex support

### PR DESCRIPTION
**Pull request by tobiasKaminsky**

### CODE REVIEW

File: .tx/config.license [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SPDX-FileCopyrightText: 2018-2023 Nextcloud GmbH and Nextcloud contributors
+SPDX-License-Identifier: GPL-3.0-or-later
```
Comment (AndyScherzinger) [inline .tx/config.license:2]: GPL or MIT, the later would match the libs license

**Review by AndyScherzinger — APPROVED** [review]

File: .tx/config.license (hunk 1 above)
Comment (tobiasKaminsky) [reply .tx/config.license:2]: Ah, right. I c&p from SSO.

**Comment by nickvergessen** [comment]
Oh, I missed this PR

**Pull request merged**